### PR TITLE
Airdrop finished screen

### PR DESCRIPTION
**Pull request by jellyfish-tom**
## What does this PR do?
Introduces airdrop finished screen:
image

## Todos:
Copy for share buttons have to be filled when provided.

- [X] Checked on iOS
- [ ] Checked on Android

### CODE REVIEW

File: src/screens/AirdropFinished/AirdropFinished.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+};
+
+export const AirdropFinished: FC<Props> = ({ route }: Props) => (
+  <ScreenTemplate header={<Header isBackArrow title={i18n.airdrop.title} />}>
+    <Text style={styles.subtitle}>{i18n.airdrop.finished.subtitle}</Text>
+    <AirdropWalletBalanceCard walletDetails={{ balance: route.params.balance, name: route.params.name }} />
+    <View style={styles.stayTunedContainer}>
+      <Text style={styles.firstLine}>
+        {i18n.formatString(i18n.airdrop.finished.stayTuned, {
+          period: 'Q1 2021',
```
Comment (marek-siemieniuk-morawski) [inline src/screens/AirdropFinished/AirdropFinished.tsx:28]: I'm wondering if it wouldn't be a good idea to keep in somewhere else, like in envs. I mean, I'm not sure, but it doesn't sound like a good design to must change a codebase every time when a new period starts. 

What do you think, @jellyfish-tom , @KedziaPawel ?

**Review by marek-siemieniuk-morawski — APPROVED** [review]

File: src/screens/AirdropFinished/AirdropFinished.tsx (hunk 1 above)
Comment (jellyfish-tom) [reply src/screens/AirdropFinished/AirdropFinished.tsx:28]: we have lots such values, currently they are kept in `consts`. I can move it there, which probably will look better, but won't change the need to update codebase with each new period.

Comment (marek-siemieniuk-morawski) [reply src/screens/AirdropFinished/AirdropFinished.tsx:28]: true. 

maybe we should ask our backend colleagues for exposing an endpoint for that or to include this information in the responses? 🤔

Comment (jellyfish-tom) [reply src/screens/AirdropFinished/AirdropFinished.tsx:28]: might be a good idea

File: src/consts/models.tsx [hunk 2]
```diff
@@ -148,6 +148,7 @@ export enum Route {
   AirdropDashboard = 'AirdropDashboard',
   AirdropThankYou = 'AirdropThankYou',
   AirdropRequirements = 'AirdropRequirements',
+  AirdropFinished = 'AirdropFinished',
```
Comment (KedziaPawel) [inline src/consts/models.tsx:176]: `AirdropFinishedWalletDetails` maybe to be more precise

File: src/screens/AirdropFinished/AirdropFinished.tsx (hunk 1 above)
Comment (KedziaPawel) [reply src/screens/AirdropFinished/AirdropFinished.tsx:28]: please move to const, for this airdrop we established that backend won't send it, later in the future we will create a more generic advanced solution with an airdrop configurator, etc.

Comment (jellyfish-tom) [reply src/screens/AirdropFinished/AirdropFinished.tsx]: done

File: src/consts/models.tsx (hunk 2 above)
Comment (jellyfish-tom) [reply src/consts/models.tsx:177]: done

**Pull request merged**